ai/core/src/unitycatalog/ai/core/utils/execution_utils.py

```diff
@@ -1,4 +1,50 @@
 import ast
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class ExecutionMode:
```

Comment:
Actually I think we don't need to unify ExecutionMode for client/databricks, it can be separate Enums in its own namespace given that they have different logics. It would be simpler if we have 
```
class ExecutionModel(str, Enum):
    SERVERLESS = "serverless"
    LOCAL = "local"

    def __str__(self):
        return self.value

    @classmethod
    def validate(cls, value):
        return value in cls.__member__.values() ....
```

Reply:
Yeah this is definitely cleaner. I'll sepearte these out and add the warning notification to the Databricks side only :)